test/e2e/scenario/agents.scenario.ts

```diff
@@ -73,7 +74,31 @@ describe('rum', () => {
   })
 
   it('should track xhr timings', async () => {
-    const timing = await makeXHRAndCollectEvent(`${browser.config.baseUrl}/ok`)
+    const timing = await makeXHRAndCollectEvent(`${serverUrl.sameOrigin}/ok`)
+    expect(timing).not.toBeUndefined()
+    expect(timing!.http.method).toEqual('GET')
+    expect((timing!.http as any).status_code).toEqual(200)
+    expectToHaveValidTimings(timing!)
+  })
+
+  it('should not track disallowed cross origin xhr timings', async () => {
+    const timing = await makeXHRAndCollectEvent(`${serverUrl.crossOrigin}/ok`)
+    expect(timing).not.toBeUndefined()
+    expect(timing!.http.method).toEqual('GET')
+    expect((timing!.http as any).status_code).toEqual(200)
+    expect(timing!.duration).toBeGreaterThan(0)
```

Comment:
could it worth to add a duration check on other tests?

Reply:
`duration` is tested by `expectToHaveValidTimings`